sandpack-react/src/utils/sandpackUtils.ts

```diff
@@ -78,6 +78,8 @@ export const getSandpackStateFromProps = (
   const files = addPackageJSONIfNeeded(
     projectSetup.files,
     projectSetup.dependencies || {},
+    projectSetup.devDependencies || {},
+    // @ts-ignore FIXME: remove this as soon as the sandpack-client typings are up-to-date
```

Comment:
if you are running the project locally, these packages should be linked together as part of the workspace and you should get the current typing for the code in `sandpack-client`